### DISCUSSION TITLE
Bootstrap Example not working in Türkiye

### DISCUSSION THREAD

**Discussion by basitiyidir** [primefaces/community#1065] [PrimeVue] [answered]
The bootstrap example on the site has not been working for a long time.

I think someone should do something.

[https://primevue.org/bootstrap/](https://primevue.org/bootstrap/)

**Note:** It does not work only for logins from Turkey. But I don't want to use **VPN** to look at the example.

**Comment by tugcekucukoglu** [comment, marked as answer]
New Stackblitz example can be found in the same page https://primevue.org/bootstrap/

Thank you.
